src/v2/view-builder/views/ov/ChallengeOktaVerifyResendPushView.js

```diff
@@ -0,0 +1,45 @@
+import { loc, createCallout } from 'okta';
+import BaseForm from '../../internals/BaseForm';
+import BaseAuthenticatorView from '../../components/BaseAuthenticatorView';
+import AuthenticatorVerifyFooter from '../../components/AuthenticatorVerifyFooter';
+
+const Body = BaseForm.extend(Object.assign(
+  {
+    className: 'okta-verify-resend-push',
+
+    title () {
+      return loc('oie.okta_verify.push.title', 'login');
+    },
+
+    save () {
+      return loc('oie.okta_verify.push.send', 'login');
+    },
+
+    showMessages () {
+      // override showMessages to display error in cases like reject push
+      // or timeout. Borrowed this logic from TerminalView.
+      const messagesObjs = this.options.appState.get('messages');
+      if (messagesObjs && Array.isArray(messagesObjs.value)) {
```

Comment:
@gowthamidommety-okta  @haishengwu-okta  feels like this is something that should go in baseForm itself. What do you guys think ?

Reply:
hmm, @gowthamidommety-okta did you get 200 response with `messages` when ResendPush ?

Reply:
Yes, it's a success response, similar to the flow in v1. That's why I'm overriding it here.. BaseForm currently shows all form messages as text even if type is ERROR.

Reply:
mock server will respond 4xx when mock file start with `error-*`.

Reply:
It may make more sense to make `showMessages` in BaseForm more generic.
Don't remember we have specific reason why only show message as text in BaseForm.
cc @pradeepdewda-okta @nikhilvenkatraman-okta

Reply:
@haishengwu-okta i loosely recall we had a long discussion about this. Earlier showMessages in the baseForm had the logic to show errors and warnings. But then @nikhilvenkatraman-okta had some different use case and then we decided to overwrite this method for the views where they needs to show these something differently. Don't remember why..  this was the PR https://github.com/okta/okta-signin-widget/pull/1274

Reply:
Seems like the reason why @nikhilvenkatraman-okta  had to override is to reduce the number of messages displayed. This is not he case here. We just can enhance the showMessages in base form. @gowthamidommety-okta  can we create a separate jira for this?